README.md

```diff
@@ -1,369 +1,137 @@
-remote: Counting objects: 4, done.
-remote: Compressing objects: 100% (2/2), done.
-remote: Total 4 (delta 3), reused 3 (delta 2), pack-reused 0
-Unpacking objects: 100% (4/4), done.
-From github.com:aviflombaum/mission-critical-application
-   bfe50fc..0ae1da2  master     -> origin/master
- * [new branch]      remote-feature-branch -> origin/remote-feature-branch
-```
+You can always move between branches with `git checkout <branch name>`. If you are currently on
+`isolating-new-feature`, you can move back to master with `git checkout master`. If the last
+branch that you switch from was master, you can also type `git checkout -` in order to move
+back to the previous branch. Then to get back to `isolating-new-feature` you can switch
+it again using `git checkout isolating-new-feature`.
 
-From within `master` (though technically what branch I was in when I typed `git fetch -a` does not matter), I executed `git fetch -a`. The last 3 lines of output are really important, let's take a closer look:
+If you have changes that are not yet committed, when you switch between branches, the untracked
+changes will follow you to the next branch. These changes must be committed with `git commit -m "<my commit message>"`
+while still on the feature or bug branch the changes belong to. When the changes on your feature
+or bug branch are committed, you'll notice that when you are on master or another branch with a
+different timeline, the commit with those changes will not be present. The master branch only has
+the code from the most recent commit relative to the master timeline or branch. The code from our
+`isolating-new-feature` is tucked away in that branch, waiting patiently in isolation from the
+rest of your code in `master` until the feature is considered complete.
 
-```
-From github.com:aviflombaum/mission-critical-application
-   bfe50fc..0ae1da2  master     -> origin/master
- * [new branch]      remote-feature-branch -> origin/remote-feature-branch
-```
+The final step of completing the `isolating-new-feature` work sprint is to merging that timeline
+into the master timeline.
 
-The first line, `From github.com:aviflombaum/mission-critical-application` is informing us which remote our `git fetch` updated from, namely, the remote repository located at: https://github.com/aviflombaum/mission-critical-application
+## Explain Merging branches
 
-When we `fetch` with git, we are asking to copy all changes on the remote to our local git repository, but not actually integrate any. The next line, `bfe50fc..0ae1da2  master     -> origin/master` is telling us that a new commit was found in `origin/master`. `origin/master` means the GitHub version of `master`. Even though git fetched a new commit from `origin/master`, it did not merge it into the local master.
+Now that you have some additions to the code that you'd like to combine back with the master set of
+code, the goal is to bring the timeline of commits that occurred on the `isolating-new-feature`
```

Comment:
OK, i'm kinda OK with timeline now, you use it a lot. You could ignore other comments as long as you bring it out as a synonym to branch earlier and explicitly. I'd still suggest cutting it and keeping with git's terminology tho.